### DISCUSSION TITLE
load ply file can't find normal and color information

### DISCUSSION THREAD

**Discussion by guyuezuntinggithub** [BrunoLevy/geogram#196] [Q&A] [answered]
Hi guys! Thanks for great work!
When I use Geogram to read a PLY file, I can't clearly see the information about normals and colors in the mesh.vertices in debug mode.
any reply will be very appreciate！

**Comment by BrunoLevy** [comment, marked as answer]
Hello !
In geogram, when loading a PLY file:
- Color information is loaded into vertex and/or facet attributes
- Vertex normals are not loaded
More information on Mesh attributes [here](https://github.com/BrunoLevy/geogram/wiki/Mesh)
